src/codegen_call_native.cpp

```
@@ -0,0 +1,181 @@
+
+    // Lazily build error function name (only for wrappings that need it)
+    auto getErrFnName = [&]() {
+        return deriveRuntimeFnName(package, "get_last_error");
+    };
+
+    // Resolve the effective runtime suffix
+    const char *rtSuffix = entry->rt_suffix ? entry->rt_suffix : entry->fn_name;
+
+    // --- Special case: variadic arity (e.g. path::join) ---
+    if (entry->arity == -1) {
+        size_t n = e.args.size();
```

Clang (14.0.6)

[warning] variable 'n' is not initialized

(cppcoreguidelines-init-variables)